x11-libs/libxcb/libxcb-1.14.ebuild

```diff
@@ -44,11 +44,10 @@ python_check_deps() {
 	has_version -b ">=x11-base/xcb-proto-1.14[${PYTHON_USEDEP}]"
 }
 
-pkg_setup() {
```

Comment:
xorg-3 has a pkg_setup too, so we need to keep an explicit definition and call both